staging/src/k8s.io/apimachinery/pkg/runtime/serializer/cbor/internal/modes/decode.go

```diff
@@ -22,6 +22,23 @@ import (
 	"github.com/fxamacker/cbor/v2"
 )
 
+var simpleValues *cbor.SimpleValueRegistry = func() *cbor.SimpleValueRegistry {
+	var opts []func(*cbor.SimpleValueRegistry) error
+	for sv := 0; sv <= 255; sv++ {
+		// Reject simple values 0-19, 23, and 32-255. The simple values 24-31 are reserved
+		// and considered ill-formed by the CBOR specification. We only accept false (20),
+		// true (21), and null (22).
+		if sv < 20 || sv == 23 || sv > 31 {
```

Comment:
supernit: Just check if 'sv' equals one of the three allowed values since it results in the same number of conditions?

Reply:
Since there is no such thing as simple values 24-31 in well-formed CBOR, the library doesn't allow you to configure them (they're always a wellformedness error). Maybe a switch statement would be more legible here.

Reply:
Changed this to a switch statement, WDYT?

Reply:
I find this easier to read. Thanks!